Sources/JavaKit/JavaObjectHolder.swift

```diff
@@ -19,13 +19,21 @@ import JavaRuntime
 /// while this instance is live.
 public class JavaObjectHolder {
   public private(set) var object: jobject?
+#if canImport(Android)
+  public var environment: JNIEnvironment {
+    try! JavaVirtualMachine.shared().environment()
+  }
+#else
   public let environment: JNIEnvironment
+#endif
 
```

Comment:
I'm not sure about such a semantic change based on platform... can would this actually do the right thing or lead to unexpected issues sometimes?

Reply:
Please see the comment in commit a2ba0e16d6255abb62598910151056f54c92340e.

Reply:
```
    JavaKit: always use ambient javaEnvironment on Android
    
    This is not a real fix, I suspect the real fix is to remove javaEnvironment as
    an instance variable on all platforms and always use the ambient environment
    (as the JNI specification clearly states the environment cannot be shared
    between threads).
    
    Works around: #157
```